plugins/dell-esrt/meson.build

```diff
@@ -0,0 +1,21 @@
+cargs = ['-DG_LOG_DOMAIN="FuPluginDellEsrt"']
```

Comment:
I think you're missing something in `plugins/meson.build` to enable this subdirectory when `plugin_uefi` is enabled